Source/Resources/MongoDB/IResource.cs

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Dolittle. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using MongoDB.Driver;
+
+namespace Dolittle.Runtime.Resources.MongoDB
+{
+    /// <summary>
+    /// Defines a system that knows about the MongoDb resource.
+    /// </summary>
+    public interface IResource
+    {
+        /// <summary>
+        /// Gets the <see cref="MongoUrl"/> for the MongoDB resource.
+        /// </summary>
+        /// <returns>The <see cref="MongoUrl"/> connection string.</returns>
+        MongoUrl GetConnectionString();
```

Comment:
Should we be using MongoUrl or ConnectionString here, are they actually the same?

Reply:
The MongoUrl is essentially the connection string

Reply:
Turns out MongoDB uses the MongoUrl under the hood: https://github.com/mongodb/mongo-csharp-driver/blob/30eded69a28ef7be8a01fdc7952dfa66a2fac8d8/src/MongoDB.Driver/MongoClientSettings.cs#L830 so this seems to be the correct thing to use.